versions/1.1.0-dev.md

```diff
@@ -365,25 +390,25 @@ paths:
           description: OK
 ```
 
-###### Overlay
+#### Overlay
 
 ```yaml
 overlay: 1.1.0
 info:
-  title: Copy contents of an existing path to a new location
+  title: Demonstrates variations of "copy" uses
   version: 1.0.0
 actions:
   - target: '$.paths["/some-items"]'
     copy: '$.paths["/items"]'
     description: 'copies recursively all elements from the "items" path item to the new "some-items" path item without ensuring the node exists before the copy'
 ```
 
-###### Result description
+#### Result description
 
 ```yaml
 openapi: 3.1.0
 info:
-  title: Example API
+  title: API with a paged collection
```

Comment:
Outdated description that we fixed in the Copy PR. Please sync your branch with the latest changes.